modules/age.nix

```diff
@@ -33,8 +35,18 @@ let
     )
     chmod ${secretType.mode} "$TMP_FILE"
     chown ${secretType.owner}:${secretType.group} "$TMP_FILE"
+
+    # see if there's been any change from the last generation
+    changes=$(${pkgs.rsync}/bin/rsync --dry-run -aHAX -i "$TMP_FILE" "$_oldPath")
```

Comment:
One way a secret could change is when the `secretType.path` is different from what it was before. Does this cover that case?

Reply:
Never mind, I figured out how to do this. Commit coming soon.